js/src/ui/Form/InputAddress/inputAddress.js

```diff
@@ -76,7 +76,7 @@ class InputAddress extends Component {
 
     const props = {};
 
-    if (!readOnly && !disabled) {
+    if (!disabled) {
```

Comment:
Yes, added here for contract params - https://github.com/ethcore/parity/commit/d16ab5eac50963c0ef9fb5bfa9e33a75f68cf914